README.md

```diff
@@ -33,17 +33,17 @@ export MY_PRISMA_API_URL="https://apix.prismacloud.io" (where x=app stack)
 export MY_PRISMA_KEY="<AccessKey>::<SecretKey>"
 ```
 
-- Using the Vscode terminal window, run a local Checkov scan:
+- Using the Vscode terminal window, run a local Checkov scan within Docker container:
 
 ```
-checkov -f yarn.lock --bc-api-key $MY_PRISMA_KEY --prisma-api-url $MY_PRISMA_API_URL  --repo-id $USER/scascan --use-enforcement-rules
+docker run --rm -v $PWD:/checkovScan -w /checkovScan bridgecrew/checkov:latest -f yarn.lock --bc-api-key $MY_PRISMA_KEY --prisma-api-url $MY_PRISMA_API_URL --repo-id $USER/scascan --use-enforcement-rules
 ```
 
-- Alternatively, if you have docker installed you may use this command:
+### Checkov installed locally
 
+If you have already installed Checkov, please follow the instructions below to complete the next steps
 ```
-docker run --rm -v $PWD:/checkovScan -w /checkovScan bridgecrew/checkov:latest -f yarn.lock --bc-api-key $MY_PRISMA_KEY --prisma-api-url $MY_PRISMA_API_URL --repo-id $USER/scascan
---use-enforcement-rules
+checkov -f yarn.lock --bc-api-key $MY_PRISMA_KEY --prisma-api-url $MY_PRISMA_API_URL  --repo-id $USER/scascan --use-enforcement-rules
```

Comment:
@SimOnPanw this is great, but my preference would be to have folks try Checkov directly first and the container method second. Can you swap the order here and I'll merge this?